kyo-offheap/shared/src/main/scala/kyo/Layout.scala

```diff
@@ -0,0 +1,67 @@
+package kyo.offheap
+
+import scala.scalanative.unsafe.*
+import scala.scalanative.unsigned.*
+
+trait Layout[A]:
+    def get(ptr: Ptr[Byte], offset: Long): A
```

Comment:
There should be no changes to the `Layout` implementation or any of the contents of `Memory.scala`.